examples/react-standalone/src/pages/index.tsx

```diff
@@ -37,7 +36,7 @@ export default function HomePage() {
     if (signClient) {
       const { uri, approval } = await signClient.connect({ requiredNamespaces: namespaces })
       if (uri) {
-        ModalCtrl.open(uri)
+        ModalCtrl.open({ uri, standaloneChains: namespaces.eip155.chains })
```

Comment:
My understanding is that in Ben's PR, it would not allow standaloneChains to pass + defaulted to line 40?

Reply:
This would be how ben uses it essentially (initialize modal on app start without standalone chains and pass them in the modal). This also triggers different behaviour in preloading of the assets i.e. instead of app start they are preloaded during open